react/Autosuggest/Autosuggest.js

```diff
@@ -152,7 +152,12 @@ export default class Autosuggest extends Component {
           ref={this.storeInputReference}
           {...allAutosuggestProps}
         />
-        {showMobileBackdrop ? <div className={styles.autosuggestBackdrop} /> : null }
+        <div
+          className={classnames({
+            [styles.autosuggestBackdrop]: true,
+            [styles.mobileBackdrop]: showMobileBackdrop
```

Comment:
we've done it because of naming clashing when it's just backdrop, now as we have transparent we can just do backdrop 👍